frontend/app/(drawer)/home.tsx

```diff
@@ -1,17 +1,27 @@
 import React from 'react';
-import { View, Dimensions, Text } from 'react-native';
+import { View, Dimensions, Text, ImageBackground } from 'react-native';
 import { useTheme } from '../../src/contexts/ThemeContext';
 import { router } from 'expo-router';
 import HeroSection from '../../src/components/HeroSection';
 import QuickActionsGrid from '../../src/components/QuickActionsGrid';
 import ModernPandaAnimation from '../../src/components/ModernPandaAnimation';
-const { width: screenWidth, height: screenHeight } = Dimensions.get('window');
 import Constants from 'expo-constants';
 
+const { width: screenWidth, height: screenHeight } = Dimensions.get('window');
+
+const pandaBackground = require('../../assets/background/fondo-panda.webp');
+
 const apiUrl = Constants.expoConfig?.extra?.API_URL ?? 'API URL not found';
 
 const HomeScreen = () => {
-  const { colors } = useTheme();
+  const { colors, setPalette, theme } = useTheme() as unknown as {
```

Comment:
What is this massive hack? 😂😅 We should not need the as unknown as... Can't we have a proper type here?